src/invidious/views/channel.ecr

```diff
@@ -20,7 +20,7 @@
     <div class="pure-u-2-3">
         <div class="channel-profile">
             <img src="/ggpht<%= URI.parse(channel.author_thumbnail).request_target %>">
-            <span><%= author %></span>
+            <span><%= author %></span><% if !channel.verified.nil? && channel.verified %> <i class="icon ion ion-md-checkmark-circle"></i><% end %>
```

Comment:
I'd recommend using an non-breakable space (`&nbsp;` in HTML) over a normal space, so it won't be inadvertently squished by the browser ;)